include/materials/ComputeIncrementalLinkStrain.h

```diff
@@ -0,0 +1,68 @@
+/*************************************************/
+/*           DO NOT MODIFY THIS HEADER           */
+/*                                               */
+/*                     MASTODON                  */
+/*                                               */
+/*    (c) 2015 Battelle Energy Alliance, LLC     */
+/*            ALL RIGHTS RESERVED                */
+/*                                               */
+/*   Prepared by Battelle Energy Alliance, LLC   */
+/*     With the U. S. Department of Energy       */
+/*                                               */
+/*     See COPYRIGHT for full restrictions       */
+/*************************************************/
+
+#ifndef COMPUTESPRINGDEFORMATION_H
+#define COMPUTESPRINGDEFORMATION_H
+
+#include "Material.h"
+#include "RankTwoTensor.h"
+/**
```

Comment:
For Doxygen to pick this up correct you need to put it just before the "class Compute..." on 31